helpers/makerpm.sh

```diff
@@ -29,36 +29,30 @@ BRANCH='HEAD'
 STABLE=0
 EXTRA=''
 
-while (( "$#" )); do
-    case "$1" in
-        "-h")
+while getopts "r:lc" opt; do
+    case ${opt} in
+        \?)
             echo "Usage: $0 [-c] [-l] [BRANCH]"
-            echo -e "\nOptions:\n\t-l: build RPM locally\n\t-c: clean build dir before starting a new build\n\t-r N: make a stable release"
+            echo -e "\\nOptions:\\n\\t-l: build RPM locally\\n\\t-c: clean build dir before starting a new build\\n\\t-r N: make a stable release"
             exit 0
             ;;
-        "-r")
+        r)
             STABLE=1
-            shift
-            if ! [[ $1 =~ ^[0-9]+$ ]] ; then
+            if ! [[ $OPTARG =~ ^[0-9]+$ ]] ; then
                echo "Error: please provide a valid PKG number" >&2; exit 1
             fi
-            PKG="$1"
-            shift
+            PKG="$OPTARG"
             ;;
-        "-l")
+        l)
             BUILD=1
-            shift
             ;;
-        "-c")
+        c)
             CLEAN=1
-            shift
-            ;;
-        *)
-            BRANCH="$1"
-            shift
             ;;
     esac
 done
+shift $((OPTIND -1))
+if [ "$1" ]; then BRANCH="$1"; fi
```

Comment:
`test "$1" && BRANCH="$1"`
Or a multi-line `if` statement, what do you think about ?